scripts/update-webapps-fluent-bit-image-to-ECR.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+# set -ex
+
+
+NAMESPACE=apps-prod
+CONTAINER=fluent-bit
+OLD_IMAGE=fluent/fluent-bit:1.1.1
+NEW_IMAGE=593291632749.dkr.ecr.eu-west-1.amazonaws.com/fluent-bit:1.1.1
+
+
+function uses_old_image() {
+    local deployment=${1:?}
+
+    kubectl -n ${NAMESPACE} get deployment "${deployment}" -o yaml | grep -q "image: ${OLD_IMAGE}"
+}
+
+
+DEPLOYMENTS=$(kubectl -n ${NAMESPACE} get deployment | grep \\-webapp | cut -d" " -f1)
+
+for deployment in $DEPLOYMENTS; do
+    echo
+    echo "Processing '${deployment}'..."
```

Comment:
```suggestion
    echo "\nProcessing '${deployment}'..."
```

This is slightly more "Bash Like"  :) 

But LGTM!

Reply:
Unfortunately don't think `echo` interprets those `\n` as you'd normally expect (I knew that but I re-tried and it didn't work, it just prints `\n` 😬 ).